contents/newsletter/choosing-technologies.md

```diff
@@ -0,0 +1,115 @@
+---
+title: What we've learned about choosing technologies
+date: 2024-10-29
+author:
+ - ian-vanagas
+featuredImage: >-
+featuredImageType: full
+tags:
+  - Engineering
+crosspost:
+  - Product engineers
+  - Founders
+  - Blog
+---
+
+From generating text to storing data to creating [ASCII drawings of cows](https://github.com/sindresorhus/cows), if you've got a problem, there is a technology out there to solve it for you.
+
+Technologies are the backbone of a product. They set the guidelines of what they can and cannot do. Choosing the right technologies makes a massive difference in whether you succeed or fail. 
+
+Unfortunately, there is no perfect way to pick a technology. Some adopt everything possible, others build anything they need themselves. Like many, we are somewhere in the middle. We aim to move fast and use technology to build something people want.
+
+Apparently, we're doing an ok job of this. Over the past year, we've shipped multiple new products like web analytics and mobile session replay on top of improving our existing products like pipelines and feature flags. Each comes with its own set of technologies. 
+
+This is what we've learned about choosing them all.
```

Comment:
Couple of thoughts here:

1. This intro feels a tiny bit on the long side. Could definitely lose the opening para.

2. I kind of wonder if we should add some context around the key technologies we've adopted? I feel like this would be useful later on when you're talking about these. Ironically, I think we could live with a longer intro if we added this context there.

Reply:
Expanded